src/main/java/nl/knaw/dans/easy/DataciteService.java

```diff
@@ -119,6 +145,18 @@ private WebResource createWebResource(String uri) {
         return client.resource(uri);
     }
 
+    private DataciteServiceException createDoiGetFailedException(int status, String cause) {
+        return new DataciteServiceException("GET doi failed: HTTP error code " + status + " and cause: " + cause);
+    }
+
+    private DataciteServiceException createDoiGetFailedException(Exception cause) {
+        return createGetFailedException("DOI", cause);
+    }
+
+    private DataciteServiceException createGetFailedException(String kind, Exception cause) {
```

Comment:
not sure if this method is necessary. `kind` is always `DOI` and is only used in the method above. Inlining and removing this method seems the best approach.